src/main/scala/tech/cryptonomic/conseil/Conseil.scala

```diff
@@ -46,46 +51,48 @@ object Conseil extends App with LazyLogging with EnableCORSDirectives with Conse
       val attributesCache = MVar[IO].empty[Map[String, (Long, List[Attribute])]].unsafeRunSync()
       val entitiesCache = MVar[IO].empty[(Long, List[Entity])].unsafeRunSync()
       lazy val tezosPlatformDiscoveryOperations =
-        TezosPlatformDiscoveryOperations(ApiOperations, attributesCache, entitiesCache, server.cacheTTL)(executionContext)
+        TezosPlatformDiscoveryOperations(ApiOperations, attributesCache, entitiesCache, server.cacheTTL)(
+          executionContext
+        )
 
       tezosPlatformDiscoveryOperations.init().onComplete {
         case Failure(exception) => logger.error("Pre-caching metadata failed", exception)
-        case Success(_) => logger.info("Pre-caching successful!")
+        case Success(_)         => logger.info("Pre-caching successful!")
       }
       lazy val platformDiscovery = PlatformDiscovery(platforms, tezosPlatformDiscoveryOperations)(tezosDispatcher)
       lazy val data = Data(platforms, tezosPlatformDiscoveryOperations)(tezosDispatcher)
 
       val route = cors() {
-        enableCORS {
-          validateApiKey { _ =>
-            logRequest("Conseil", Logging.DebugLevel) {
-              tezos.route ~
-              AppInfo.route
+          enableCORS {
```

Comment:
I told you at some point that auto code rearrangement will break those routes 😉

Reply:
by break I meant rearrange it the different way than you proposed in one of my PRs - maybe scalafmt has something special prepared for the akka routes?

Reply:
I couldn't figure out if there was a substantial change when I reformatted those.
In any case, I simply thought that we have `//format: off` as our weapon for critical sections where formatting is an art more than a science 😄